.github/workflows/build.yml

```diff
@@ -392,14 +392,37 @@ jobs:
                   name: Eden Microchain
                   path: build
 
+            - name: Download Ephemeral Chain Runners
+              if: steps.filter.outputs.src == 'true'
+              uses: actions/download-artifact@v2
+              with:
+                  name: Ephemeral Eden Chains Runners
+                  path: build
+
+            - name: Download clsdk
```

Comment:
This may get tricky. Optimal solution, if it's possible with GH actions:
* If build-cpp builds, then grab the clsdk artifact
* If build-cpp doesn't build, then grab the clsdk from somewhere else. The question is, where?